docs/ruleset_functions/get_zone_conditioning_category.md

```diff
@@ -17,76 +21,96 @@ Logic:
+
+- For each zone in RMR: `for zone in RMR...zones:`
 
-- For each building segment in the RMR: `for building_segment in RMR.building.building_segments:`  
+  - Get zone total floor area: `zone_area = SUM(space.floor_area for space in zone.spaces)`
 
-  - To determine eligibility for directly conditioned (heated or cooled) and semi-heated zones, for each HVAC system in building segment: `for hvac_system in building_segment.heating_ventilation_air_conditioning_systems:`  
+  - For each terminal serving zone: `for terminal in zone.terminals:`
 
-    - Check if the system meets the criteria for serving directly conditioned (heated or cooled) zones, save all zones served by the system as directly conditioned: `if ( hvac_system.simulation_result_sensible_cool_capacity >= CAPACITY_THRESHOLD ) OR ( hvac_system.simulation_result_heat_capacity >= system_min_heating_output ):  for zone in hvac_system.zones_served: directly_conditioned_zones.append(zone)`  
+    - Get HVAC system connected to terminal: `hvac_sys = terminal.served_by_heating_ventilation_air_conditioning_systems` (Note XC, will there be more than one HVAC system connecting to a terminal?)
 
-    - Else check if the system meets the criteria for serving semi-heated zones, save all zones served by the system as semi-heated: `else if hvac_system.simulation_result_heat_capacity >= CAPACITY_THRESHOLD: for zone in hvac_system.zones_served: semiheated_zones.append(zone)`  
+    - Add central cooling capacity per floor area to zone capacity dictionary: `zone_capacity_dict[zone.id]["SENSIBLE_COOLING"] += hvac_cool_capacity_dict[hvac_sys.id]`
 
-  - To determine eligibility for indirectly conditioned zones, for each zone in building_segment: `for zone in building_segment.zones:`  
+    - Add central heating capacity per floor area to zone capacity dictionary: `zone_capacity_dict[zone.id]["HEATING"] += hvac_heat_capacity_dict[hvac_sys.id]`
 
-    - If zone is not directly conditioned (heated or cooled): `if zone not in directly_conditioned_zones:`  
+    - Check if terminal has heating capacity, add to zone capacity dictionary: `if terminal.heat_capacity: zone_capacity_dict[zone.id]["HEATING"] += terminal.heat_capacity / zone_area` (Note XC, it seems terminal does not have cooling capacity. Might need cooling capacity at the terminal level for equipment like active beams.)
 
-      - If any space in zone is atrium, zone is indirectly conditioned:: `if ( ( space.lighting_space_type == "ATRIUM_LOW_MEDIUM" ) OR ( space.lighting_space_type == "ATRIUM_HIGH" ) for space in zone.spaces ): indirectly_conditioned_zones.append(zone)`  
+  - Check if zone meets the criteria for directly conditioned (heated or cooled) zone, save zone as directly conditioned: `if ( zone_capacity_dict[zone.id]["SENSIBLE_COOLING"] >= CAPACITY_THRESHOLD ) OR ( zone_capacity_dict[zone.id]["HEATING"] >= system_min_heating_output ): directly_conditioned_zones.append(zone.id)`
 
```

Comment:
Per comment on line #17, CAPACITY_THRESHOLD will be replaced with System_min_sens_cool_output